2025-03-05 09:13:08,115 INFO    MainThread:21278 [wandb_setup.py:_flush():67] Current SDK version is 0.19.7
2025-03-05 09:13:08,115 INFO    MainThread:21278 [wandb_setup.py:_flush():67] Configure stats pid to 21278
2025-03-05 09:13:08,115 INFO    MainThread:21278 [wandb_setup.py:_flush():67] Loading settings from /home/shoaiboffice/.config/wandb/settings
2025-03-05 09:13:08,115 INFO    MainThread:21278 [wandb_setup.py:_flush():67] Loading settings from /home/shoaiboffice/Personal/Personal projects/Deepseek-r1-bangla-distill-dataset/code/DeepSeek-r1-Bangla-Reasoning-Data/wandb/settings
2025-03-05 09:13:08,115 INFO    MainThread:21278 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-05 09:13:08,115 INFO    MainThread:21278 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /home/shoaiboffice/Personal/Personal projects/Deepseek-r1-bangla-distill-dataset/code/DeepSeek-r1-Bangla-Reasoning-Data/wandb/run-20250305_091308-14wyjgop/logs/debug.log
2025-03-05 09:13:08,116 INFO    MainThread:21278 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /home/shoaiboffice/Personal/Personal projects/Deepseek-r1-bangla-distill-dataset/code/DeepSeek-r1-Bangla-Reasoning-Data/wandb/run-20250305_091308-14wyjgop/logs/debug-internal.log
2025-03-05 09:13:08,116 INFO    MainThread:21278 [wandb_init.py:init():761] calling init triggers
2025-03-05 09:13:08,116 INFO    MainThread:21278 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-05 09:13:08,116 INFO    MainThread:21278 [wandb_init.py:init():784] starting backend
2025-03-05 09:13:08,116 INFO    MainThread:21278 [wandb_init.py:init():788] sending inform_init request
2025-03-05 09:13:08,123 INFO    MainThread:21278 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-03-05 09:13:08,124 INFO    MainThread:21278 [wandb_init.py:init():803] backend started and connected
2025-03-05 09:13:08,125 INFO    MainThread:21278 [wandb_init.py:init():896] updated telemetry
2025-03-05 09:13:08,132 INFO    MainThread:21278 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-03-05 09:13:08,942 INFO    MainThread:21278 [wandb_init.py:init():995] starting run threads in backend
2025-03-05 09:13:09,137 INFO    MainThread:21278 [wandb_run.py:_console_start():2377] atexit reg
2025-03-05 09:13:09,138 INFO    MainThread:21278 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-05 09:13:09,138 INFO    MainThread:21278 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-05 09:13:09,138 INFO    MainThread:21278 [wandb_run.py:_redirect():2317] Redirects installed.
2025-03-05 09:13:09,143 INFO    MainThread:21278 [wandb_init.py:init():1037] run started, returning control to user process
2025-03-05 16:36:54,188 WARNING MsgRouterThr:21278 [router.py:message_loop():77] message_loop has been closed
Traceback (most recent call last):
  File "/home/shoaiboffice/Personal/Personal projects/Deepseek-r1-bangla-distill-dataset/code/DeepSeek-r1-Bangla-Reasoning-Data/venv/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 28, in _read_message
    resp = self._sock_client.read_server_response(timeout=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/shoaiboffice/Personal/Personal projects/Deepseek-r1-bangla-distill-dataset/code/DeepSeek-r1-Bangla-Reasoning-Data/venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 285, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/shoaiboffice/Personal/Personal projects/Deepseek-r1-bangla-distill-dataset/code/DeepSeek-r1-Bangla-Reasoning-Data/venv/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 270, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/shoaiboffice/Personal/Personal projects/Deepseek-r1-bangla-distill-dataset/code/DeepSeek-r1-Bangla-Reasoning-Data/venv/lib/python3.12/site-packages/wandb/sdk/interface/router.py", line 70, in message_loop
    msg = self._read_message()
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/shoaiboffice/Personal/Personal projects/Deepseek-r1-bangla-distill-dataset/code/DeepSeek-r1-Bangla-Reasoning-Data/venv/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 30, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-03-05 16:36:54,215 INFO    MsgRouterThr:21278 [mailbox.py:close():115] Closing mailbox, abandoning 1 handles.
